src/pymortests/mu_derivatives.py

```diff
@@ -0,0 +1,75 @@
+# This file is part of the pyMOR project (http://www.pymor.org).
+# Copyright 2013-2019 pyMOR developers and contributors. All rights reserved.
+# License: BSD 2-Clause License (http://opensource.org/licenses/BSD-2-Clause)
+
+from pymortests.base import runmodule
+import pytest
+
+import numpy as np
+
+from pymor.parameters.functionals import ProjectionParameterFunctional, ExpressionParameterFunctional
+from pymor.operators.constructions import LincombOperator, ZeroOperator
+from pymor.basic import NumpyVectorSpace
+
+pf = ProjectionParameterFunctional('mu', (2,), (0,))
+
+dict_of_partial_derivatives = {'mu': ['100', '2'], 'nu': 'cos(nu)'}
+
+epf = ExpressionParameterFunctional('100 * mu[0] + 2 * mu[1] + sin(nu)',
+                                    {'mu': (2,), 'nu': ()},
+                                    'functional_with_derivative',
+                                    dict_of_partial_derivatives)
```

Comment:
Initialization code for a test should not be executed on module import. It would better to either move this into the test functions or into a fixture.